### DISCUSSION TITLE
I'm spawning as a bird in my dedicated server

### DISCUSSION THREAD

**Discussion by SergDLeon** [Vdauphin/HeartsAndMinds#1450] [Question and answer]
Hi,  I love your game mode but every time I upload it on my dedicated server I spawn as a bird, but when I play it on editor I spawn as my character. Can you help me?

**Comment by Vdauphin** [comment]
Hello,

>  Can you help me?

How?

Cheers

**Reply by Souza-R** [reply]
i had this problem once! it is related to the map .. i solved reducing the amount of vehicles btc_veh

**Reply by Souza-R** [reply]
it only happened on umb_colombia map

**Reply by Playmakinggg** [reply]
Can you further explain this? I am running into this issue.

**Reply by BadDad1776** [reply]
did any of you ever figure this out... i can not get it to boot on my dedicated server but if i run as a local server on my pc it boots fine

everytime i boot on the dedicated server and slot in to a player slot i spawn as a bird

**Reply by Souza-R** [reply]
Hi! First I’d like to say that my English isn’t very good and that I haven’t played Arma 3 for a long time.

I simply reduced the number of vehicles at the base. I’m not sure how the current version works — whether the vehicles have IDs or if you just place them in the respawnable folder in the EDEN Editor. The solution I found was to reduce the number of vehicles, both the ones without respawn and the ones with respawn.

I recommend testing with a small number of vehicles and gradually increasing it.

**Reply by BadDad1776** [reply]
The bird issue is happening on the base Altis mission file... no changes have been made
